connectors/sources/salesforce.py

```diff
@@ -405,15 +405,22 @@ async def get_cases(self):
         async for records in self._yield_non_bulk_query_pages(query):
             case_feeds_by_case_id = {}
             if await self._is_queryable("CaseFeed") and records:
-                case_ids = [x.get("Id") for x in records]
-                case_feeds = await self.get_case_feeds(case_ids)
+                all_case_ids = [x.get("Id") for x in records]
+                case_ids_list = [
+                    all_case_ids[i : i + 800] for i in range(0, len(all_case_ids), 800)
+                ]
```

Comment:
Question: where does 800 come from? Is it the upper limit for query size?

Reply:
`IN` clause in SOQL query can't exceed more than 1000 ids or 20k characters whatever is first met. So, for the safer side i kept 800. while testing the E2E, i faced an error so i've changed this.